src/index.ts

```diff
@@ -37,7 +37,7 @@ export async function resolve(
   const { parentURL = baseURL } = context;
 
   let forceRelative = false;
-  if (TSConfig?.paths) {
+  if (TSConfig && TSConfig.paths) {
```

Comment:
This is not "GitHub Actions doesn't like nullish coalescing", but your tsconfig is targeting ESNext so that tsc emits `?.` into JavaScript.
https://github.com/K-FOSS/TS-ESNode/blob/cfbf2f3df1c981d43d4b6004478d46d72995eec3/tsconfig.json#L3

Then, Node 14.x supports nullish coalescing, so it works well. Node 13.x does not support this syntax so it errors.
https://kangax.github.io/compat-table/es2016plus/

The correct solution is to set a lower target in tsconfig, instead of avoiding new syntax.
I usually use ES2019 target, which seems to work well.

Reply:
Yeah, I thought 13.9+ included nullish coalescing and optional chaining. Everything was working locally but errored on GitHub Actions. Didn't realize it was made stable on 14. Really thought it was stable 13.9. But... as I write this I'm remembering the harmony option I was using in December.

Reply:
`--harmony-optional-chaining` guess it is enabled on Node.JS 14 by default. `node --v8-options` is fun to play with future V8/Node.JS/ESNext features.

Reply:
@yoursunny Should I just set TS-ESNode to target Node.JS `>= 14`. I mean who is using TS-ESNode without latest everything anyway?